apps/twap/src/app/orderbook/order.ts

```diff
@@ -0,0 +1,126 @@
+import { Column, PrimaryColumn, Entity } from 'typeorm';
+import {
+  bigIntToString,
+  bufferToString,
+  stringToBigInt,
+  stringToBuffer,
+} from '../utils/transformers';
+
+@Entity({ name: 'orders' })
+export class Order {
+  @PrimaryColumn('bytea', {
+    name: 'uid',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  uid: string;
+
+  @Column('bytea', {
+    name: 'owner',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  owner: string;
+
+  @Column('timestamp with time zone', { name: 'creation_timestamp' })
+  creationTimestamp: Date;
+
+  @Column('bytea', {
+    name: 'sell_token',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  sellToken: string;
+
+  @Column('bytea', {
+    name: 'buy_token',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  buyToken: string;
+
+  @Column('numeric', { name: 'sell_amount' })
+  sellAmount: number;
+
+  @Column('numeric', { name: 'buy_amount' })
+  buyAmount: number;
```

Comment:
Shouldn't the amounts be bigints as well?

Reply:
![image](https://github.com/cowprotocol/cow-web-services/assets/5172699/9df1d244-6453-4f2a-be88-92780444dea0)

DB definition on this is numeric, so not particularly certain. As opposed to valid_to, which is bigint.